### PR TITLE
Add env

### PR DESCRIPTION
**Pull request by Oleg-Decik**

### CODE REVIEW

**Review by ArtemHurin — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -51,6 +51,11 @@
         </plugins>
     </build>
     <dependencies>
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
```
Comment (ostret) [inline pom.xml:55]: do you need it ?

File: src/main/java/data/UserRepository.java [hunk 2]
```diff
@@ -25,15 +25,15 @@ public UserRepository() {
...
+                System.getProperty ( "mariaEmail" ),
+                System.getProperty ( "masterPass" ),
                 "03/03/2001",
                 Boolean.TRUE,
                 Boolean.TRUE);
 
         admin = new User();
-        admin.setEmail("admin@gmail.com");
-        admin.setPassword("admin");
+        admin.setEmail(System.getProperty ( "adminEmail" ));
```
Comment (ostret) [inline src/main/java/data/UserRepository.java:35]: move all getProperty to helper class
make sure tests fail gracefully if property if not set
extract all required properties in beforeSuite
throw new SkipException("Required property is missing" )  if it is not set

File: pom.xml (hunk 1 above)
Comment (Oleg-Decik) [reply pom.xml:55]: actualy not)

File: src/main/java/data/UserRepository.java (hunk 2 above)
Comment (Oleg-Decik) [reply src/main/java/data/UserRepository.java:35]: it wasn`t my task)we will correct it)

**Pull request closed without merging**